### ISSUE TITLE
Uploads fail on IE 11

### ISSUE THREAD

**Issue by musemega** [tomitrescak/meteor-uploads#209]
Trying to upload any image file on Windows 10 (IE ver: 11; update version 11.0.25)  and IE immediately throws the "Internet Explorer has stopped working" dialog.

I've tried this on several machines and all exhibit the same behavior.

Could you take a look?

**Comment by tomitrescak** [comment]
You are right ... I have never tested it before. Did not know that people are still using IE :) Unfortunatelly I do not have time to debug this now ... if you manage to find time I would be greatfull. Plugin has been around for more than a year now and this is a first report, so it is not a stressing issue for now. Sorry :/

**Comment by musemega** [comment]
No prob, and IE -  I know!  Win10 is bringing a whole new set of IE (and Edge) users.  Great…I’ll take a look and see what I can do.SteveOn Feb 1, 2016, at 11:10 PM, Tomas Trescak <notifications@github.com> wrote:You are right ... I have never tested it before. Did not know that people are still using IE :) Unfortunatelly I do not have time to debug this now ... if you manage to find time I would be greatfull. Plugin has been around for more than a year now and this is a first report, so it is not a stressing issue for now. Sorry :/—Reply to this email directly or view it on GitHub.


  
  




muse megav: 206.395.3100 c: 425.233.0981 l : www.linkedin.com/in/stevenejackson/

**Comment by dwayneholmberg** [comment]
Is this just when using the upload_bootstrap or upload_semanticUI components? I just set up meteor-uploads using the custom template from the README, and it's working fine on Windows 10 in Edge, IE11 and IE10. I haven't tried with the build in components, mind you. 

Very nice package, by the way.

**Comment by musemega** [comment]
It is the upload_semanticUI component.  I haven't had the cycles to take a look yet, but it's on the task list.

**Comment by dwayneholmberg** [comment]
Thanks for the clarification! I'm working with the bootstrap component at the moment, but I'll keep on eye open on the semantic side if I stumble over there. Cheers.
